cmd/verify_metadata.go

```diff
@@ -144,41 +150,137 @@ func verifyMetadata(cmd *cobra.Command, args []string) error {
         return err
     }
 
+    if metadata_type == "premiuminfo" {
+        return verifyPremiumMetadata(cmd, serviceAccountKeyFilePath, destDir, artifactPath, language, packageID, version, disableCertificateVerification, disableDeletes)
+    }
+
+    return verifyNONPremiumMetadata(cmd, serviceAccountKeyFilePath, destDir, language, packageID, version, metadata_type, disableCertificateVerification, disableDeletes)
+}
+
+func verifyPremiumMetadata(cmd *cobra.Command, serviceAccountKeyFilePath, destDir, artifactPath, language, packageID, version string, disableCertificateVerification, disableDeletes bool) error {
```

Comment:
there are lot of inputs fields, we can make a struct and take it as an input.

Reply:
you can also add a comment, on how you are verifying the premium metadata.

Reply:
done